veille0405 2026 fr & en (#881)
Orange-OpenSource/a11y-guidelines: src/en/articles/watch-april-may-2026.md

- Creating Accessible Digital Documents, Introduction: https://tetralogical.com/blog/2026/05/13/creating-accessible-documents/
- Auditing at the end of the cycle is not the solution, shift left: https://buttondown.com/access-ability/archive/you-can-t-audit-your-way-into-accessibility/
- A11y overlays, that's bad, let's talk about it again: https://the-word.news/2026/0402/accessibility-overlays-create-problematic-quick-fixes/
- One year after the EAA, reflections: https://gbbns.co/journal/accessibility-problem-isnt-design/
- We need people dedicated to A11y: https://gbbns.co/journal/design-brief-that-never-mentioned-disabled-people/
- A11y debt compared to UX debt, the cost is much higher, read about it: https://www.accessux.com.au/blog/design-debt-vs-accessibility-debt-why-fixing-it-early-saves-millions/
- No secret, A11y from the start, interesting: https://annaecook.com/writing/2026/ai-doesnt-fix-accessible-systems-it-depends-on-them

## Testing Accessibility

- Criterion 2.5.8 Target Size: https://tarnoff.info/2026/04/06/A11y-101-2-5-8-target-size/

> - Further testing with Playright than Axe or Lighthouse: https://247webdevs.blogspot.com/2026/05/why-keyboard-users-cant-scroll-your.html

Reviewer: ```suggestion
- Further testing with Playwright than Axe or Lighthouse: https://www.davidmello.com/software-testing/test-automation/playwright-accessibility-testing-axe-lighthouse-limitations
```

> - Wcag 3.0 and Contrast Ratio Calculation: https://adrianroselli.com/2026/04/wcag3-contrast-as-of-april-2026.html

Reviewer: ```suggestion
- WCAG 3.0 and Contrast Ratio Calculation: https://adrianroselli.com/2026/04/wcag3-contrast-as-of-april-2026.html
```

---
title: "#A11y technical watch, April-May 2026"
abstract: "#A11y anthology, April-May 2026"

> date: "2025-06-08"

Reviewer: ```suggestion
date: "2026-06-08"
```

## Artificial Intelligence (AI)

- A GitHub AI Agent for A11y, Experimentation Feedback, Not Bad: https://github.blog/ai-and-ml/github-copilot/building-a-general-purpose-accessibility-agent-and-what-we-learned-in-the-process/
- AI and A11y: An Overview: https://annaecook.com/writing/2026/ai-doesnt-fix-accessible-systems-it-depends-on-them
- Another, more comprehensive overview in two parts:

 - https://onsman.com/the-impact-of-ai-on-digital-accessibility-part-i/
 - https://onsman.com/the-impact-of-ai-on-digital-accessibility-part-ii/
- An Agent Skill (currently under development) for iOS developers, worth testing: https://github.com/dadederk/iOS-Accessibility-Agent-Skill
- Don't rely on AI to generate your image alternatives: https://silktide.com/blog/the-downsides-of-ai-alt-text/
- A11y-specific AI chatbot by WebAim: https://webaim.org/AIMee/
- Stereotypes are cognitive biases: https://A11ywithdiana.substack.com/p/did-you-ever-think-about-how-the
- Testing keyboard accessibility with AI: https://html5accessibility.com/stuff/2026/04/29/keyboard-testing-with-the-power-of-ai/

> - An evaluation of LLM and A11y: https://microsoft.github.io/A11y-llm-eval-report/

Reviewer: Link in 404.

> - An evaluation of LLM and A11y: https://microsoft.github.io/A11y-llm-eval-report/

Reviewer: ```suggestion
- An evaluation of LLM and A11y: https://microsoft.github.io/a11y-llm-eval-report/
```

- Google's Modern Web Guidance, to avoid: https://adrianroselli.com/2026/05/maybe-dont-rely-on-googles-modern-web-guidance.html

## Resources

> - Improvements for 3D on the web and A11y: https://cerovac.com/A11y/2026/04/new-hope-for-better-accessibility-of-3d-experiences-on-the-web/

Reviewer: Link in 404.

> - Improvements for 3D on the web and A11y: https://cerovac.com/A11y/2026/04/new-hope-for-better-accessibility-of-3d-experiences-on-the-web/

Reviewer: ```suggestion
- Improvements for 3D on the web and A11y: https://cerovac.com/a11y/2026/04/new-hope-for-better-accessibility-of-3d-experiences-on-the-web/
```
